docs/baseband/tutorials/getting_started.rst

```diff
@@ -0,0 +1,388 @@
+
+    >>> fh
+    <VDIFStreamReader name=... offset=24
+        frames_per_second=1600, samples_per_frame=20000,
+        sample_shape=SampleShape(nthread=8),
+        complex_data=False, bps=2, edv=3, station=65532,
+        (start) time=2014-06-16T05:56:07.000000000>
+
+The ``offset`` gives the current location of the sample file pointer - it's at
+``24`` since we have just read in 24 (complete) samples.  If we called
+``fh.read(12)`` again we would get the next 12 samples.  If we instead called 
+``fh.read()``, it would read from the pointer's *current* position to the end
+of the file.  If we wanted all the data in one array, we would move the file
+pointer back to the start of file, using ``fh.seek``, before reading::
+
+    >>> fh.seek(0)      # Seek to sample 0.  Seek returns its offset in counts.
+    0
+    >>> d_complete = fh.read()
+    >>> d_complete.shape
+    (40000, 8)
+
+We can also move the pointer with respect to the end of file by passing ``2``
+as a second argument::
+
+    >>> fh.seek(-100, 2)    # Second arg is 0 (start of file) by default.
+    39900
+    >>> d_end = fh.read(100)
+    >>> np.array_equal(d_complete[-100:], d_end)
+    True
+
+``-100`` means 100 samples before the end of file, so ``d_end`` is equal to
+the last 100 entries of ``d_complete``.  Baseband only keeps the most recently
+accessed data frame in memory, making it possible to analyze (normally large)
+files through selective decoding using ``seek`` and ``read``.
+
+.. note::
+
+    Cation should be used when decoding large blocks of data using
+    ``fh.read``.  For typical files, the resulting arrays are far too
+    large to hold in memory.
+
+To determine where the pointer is located, we use ``fh.tell()``::
+
+    >>> fh.tell()
+    40000
+    >>> fh.close()
+
+Seeking and Telling in Time With the Sample Pointer
+---------------------------------------------------
+
+We can use ``seek`` and ``tell`` with units of time rather than samples.  To do
+this with ``tell``, we can pass an appropriate `astropy.units.Unit` object to
+its optional ``unit`` parameter::
+
+    >>> fh = vdif.open(SAMPLE_VDIF, 'rs')
+    >>> fh.seek(40000)
+    40000
+    >>> fh.tell(unit=u.ms)
+    <Quantity 1.25 ms>
+
+Passing the string ``'time'`` reports the pointer's location in absolute time::
+
+    >>> fh.tell(unit='time')
+    <Time object: scale='utc' format='isot' value=2014-06-16T05:56:07.001250000>
+
+We can also pass an absolute `astropy.time.Time`, or a positive or negative time
+difference `~astropy.time.TimeDelta` or `astropy.units.Quantity` to ``seek``. 
+If the offset is a `~astropy.time.Time` object, the second argument to seek is
+ignored.
+
+::
+
+    >>> from astropy.time.core import TimeDelta
+    >>> from astropy.time import Time
+    >>> fh.seek(TimeDelta(-5e-4, format='sec'), 2)  # Seek -0.5 ms from end.
+    24000
+    >>> fh.seek(0.25*u.ms, 1)  # Seek 0.25 ms from current position.
+    32000
+    >>> # Seek to time index 2014/06/16 5:56:07.001125
```

Comment:
Just write `Seek to specific time` (It seems strange to use a different time format here).